apps/condo/domains/organization/integrations/sbbol/sync/syncBankAccounts.js

```diff
@@ -63,7 +64,26 @@ const _syncBankAccounts = async (accounts, organization) => {
                     organization: { connect: { id: organization.id } },
                 }
             )
-            logger.info('Created BankAccount', { bankAccount: { id: account.number, organization: { id: organization.id, name: organization.name } } })
+            logger.info({ msg: 'Created BankAccount', bankAccount: { id: account.number, organization: { id: organization.id, name: organization.name } } })
+        } else {
+            const haveSBBOLIntegrationContext = get(foundAccount, 'integrationContext.integration.id') === integration.id
+
+            if (!haveSBBOLIntegrationContext) {
+                logger.info({ msg: 'Found account have`nt integrationContext' })
+
+                const createdBankIntegrationContext = await BankIntegrationContext.create(context, {
+                    ...dvSenderFields,
+                    integration: { connect: { id: integration.id } },
+                    organization: { connect: { id: organization.id } },
+                })
+
+                await BankAccount.update(context, foundAccount.id, {
+                    integrationContext: { connect: { id: createdBankIntegrationContext.id } },
+                    ...dvSenderFields,
+                })
+
+                logger.info({ msg: `Connected BankIntegrationContext { id: ${createdBankIntegrationContext.id} } to BankAccount { id: ${foundAccount.id} }` })
```

Comment:
Could you please add 'integration : { name }' to log record

Reply:
Logger will already mark it as sbbol/syncBankAccounts